overlord/managers_test.go

```diff
@@ -1142,6 +1153,17 @@ type: kernel`
 	chg := st.NewChange("install-snap", "...")
 	chg.AddAll(ts)
 
+	// run, this will wait trigger a wait for the restart
+	st.Unlock()
+	ms.o.Settle(250 * time.Millisecond)
```

Comment:
isn't the timeout here a bit short?

Reply:
Aha, sorry, updated now to use the normal settle timeout.